main/views.py

```diff
@@ -169,34 +169,33 @@ def github_code_to_member(code, ohmember):
             'client_id': settings.GITHUB_CLIENT_ID,
             'client_secret': settings.GITHUB_CLIENT_SECRET
         }
-        req = requests.post(
-            'https://github.com/login/oauth/access_token'.format(
-                settings.OPENHUMANS_OH_BASE_URL),
-            data=data
-        )
+        # Add headers telling Github's API that we want a JSON response (instead of plaintext)
+        headers = {'Accept': 'application/json'}
+        # Get the access_token from the code
+        req = requests.post('https://github.com/login/oauth/access_token',
+                            data=data, headers=headers)
```

Comment:
Here we're just adding the headers so that we get a response in JSON which is easier to parse (normally, the response is sent as a string). See [here](https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/#response) for Github's API docs on token responses.